src/Umbraco.Core/Composing/TypeFinder.cs

```diff
@@ -49,13 +49,25 @@ public class TypeFinder : ITypeFinder
     private string[]? _assembliesAcceptingLoadExceptions;
     private volatile HashSet<Assembly>? _localFilteredAssemblyCache;
 
+    [Obsolete("Please use constructor that takes additionalExlusionAssemblies, scheduled for removal in v14")]
     public TypeFinder(ILogger<TypeFinder> logger, IAssemblyProvider assemblyProvider, ITypeFinderConfig? typeFinderConfig = null)
     {
         _logger = logger ?? throw new ArgumentNullException(nameof(logger));
         _assemblyProvider = assemblyProvider;
         _typeFinderConfig = typeFinderConfig;
     }
```

Comment:
The old constructor should just call the new constructor

Reply:
```suggestion
    [Obsolete("Please use the constructor taking all parameters. This constructor will be removed in V14.")]
    public TypeFinder(ILogger<TypeFinder> logger, IAssemblyProvider assemblyProvider, ITypeFinderConfig? typeFinderConfig = null)
        : this(logger, assemblyProvider, null, typeFinderConfig)
    { }
```